src/app/drivingLicence/controllers/root.js

```diff
@@ -1,19 +1,73 @@
 const { Controller: BaseController } = require("hmpo-form-wizard");
 
+const {
+  API: {
+    BASE_URL,
+    PATHS: { PERSON_INFO }
+  }
+} = require("../../../lib/config");
+
 class RootController extends BaseController {
   async saveValues(req, res, next) {
-    const sharedClaims = req.session?.shared_claims;
+    req.sessionModel.reset();
+    req.sessionModel.set("isAuthSourceRoute", false);
 
-    if (sharedClaims) {
-      if (sharedClaims?.names?.length > 0) {
-        req.journeyModel.set("surname", sharedClaims.names[0]?.familyName);
-        req.journeyModel.set("givenNames", sharedClaims.names[0]?.givenNames);
-      }
+    let personInfoApiResponse = {};
 
-      if (sharedClaims?.dateOfBirths?.length > 0) {
-        req.journeyModel.set("dateOfBirth", sharedClaims.dateOfBirths[0]);
+    const headers = {
+      session_id: req.session?.tokenId
+    };
+
+    if (process.env.AUTH_SOURCE_ENABLED === "true") {
+      //axios.post to new personInfo endpoint, put licence details into req.session.shared_claims shared claims
+      personInfoApiResponse = await req.axios.post(
+        `${BASE_URL}/${PERSON_INFO}`,
+        {},
+        { headers: headers }
+      );
+    }
+    if (personInfoApiResponse.data.drivingPermit) {
```

Comment:
This should be in the above feature flag if.

It also needs to handle a failure of personInfoApiResponse not being returned and .data.drivingPermit not being present.